### DISCUSSION TITLE
Profile announcing you updated a merge request is very misleading

### DISCUSSION THREAD

**Discussion by tilkinsc** [community/community#18310] [Other Feature Feedback, Questions, & Ideas]
# problem 
I was trawling through one of my Forker's profile after they pushed some commits to change a PR that was then merged. This is very misleading and it appears as though they directly commited to the repo. When all they did was they got a PR merged.

![image](https://user-images.githubusercontent.com/7494772/173163026-c6980181-180f-4f03-bb5a-f66b1ca6f510.png)

# resolution
Be a lot more specific about what went on.

**Comment by PROxZIMA** [comment]
Contribution activity does show details about that commit which is just below the "Created commits in repository" section. You can click on the "2 commits" to see the author's commit for the current month in the respective repository. 

![image](https://user-images.githubusercontent.com/43103163/173187201-cfbbd850-e465-44d7-acbb-38bacf8c7e6f.png)

**Reply by tilkinsc** [reply]
It's misleading. They didn't add commits directly to the project. They had a PR merged. They didn't contribute directly to the project as in a push to a branch. I had to check perms on my repo because I wasn't sure they actually directly added commits.
